### ISSUE TITLE
Add conversions from `SwapCommitment` to `tct::Commitment`

### ISSUE THREAD

**Issue by hdevalence** [penumbra-zone/penumbra#1714]
After #1713, we'll want to implement conversions from `SwapCommitment` to `tct::Commitment`, similar to the ones for `note::Commitment`, so we can insert swap commitments into the TCT.

**Comment by redshiftzero** [comment]
Closing in favor of using the `tct::Commitment` type directly for swaps, see https://github.com/penumbra-zone/penumbra/pull/1723#issuecomment-1343242889.
